nba_pagerank.py

```diff
@@ -1,32 +1,79 @@
 import csv
-
-
-class NbaPagerank(object):
+import numpy as np
+
+class PageRank(object):
+    def __init__(self, M, R):
```

Comment:
maybe more descriptive variable names?
```suggestion
    def __init__(self, transition_matrix, ranks):
```

also, #5 and #6  for adding static typechecker and linter